meta-sgl-core/conf/distro/sgl.conf

```
@@ -0,0 +1,74 @@
+DISTRO = "sgl"
+DISTRO_NAME = "Space Grade Linux"
+DISTRO_VERSION = "0.1"
+DISTRO_CODENAME = "wone"
+SDK_VENDOR = "-sglsdk"
+SDK_VERSION = "${@d.getVar('DISTRO_VERSION').replace('snapshot-${METADATA_REVISION}', 'snapshot')}"
+SDK_VERSION[vardepvalue] = "${SDK_VERSION}"
+
+MAINTAINER = "non Yet"
+
+TARGET_VENDOR = "-sgl"
+
+LOCALCONF_VERSION = "2"
+
+# Override these in poky based distros
+SGL_DEFAULT_DISTRO_FEATURES = "ptest"
+SGL_DEFAULT_EXTRA_RDEPENDS = "packagegroup-core-boot"
+
+DISTRO_FEATURES ?= "${DISTRO_FEATURES_DEFAULT} ${SGL_DEFAULT_DISTRO_FEATURES}"
```

```suggestion
DISTRO_FEATURES ?= "${SGL_DEFAULT_DISTRO_FEATURES}"
```
I think there was some talking around choosing not to use the OE core defaults here and just being very explicit in our `SGL_DEFAULT_DISTRO_FEATURES` var. Thoughts?